app/views/employment/CheckYourBenefitsViewEOY.scala.html

```diff
@@ -37,8 +37,8 @@
 
 @hiddenText(item:String) = @{Some(messages(s"$item.hiddenText.${if(user.isAgent) "agent" else "individual"}"))}
 
-@showRows(item: String, value: BigDecimal) = @{
-    summaryListRow(HtmlContent(messages(item)),HtmlContent(bigDecimalCurrency(value.toString)),actions = Seq((controllers.employment.routes.CheckYourBenefitsController.show(taxYear, employmentId), messages("common.change"), hiddenText(item))), keyClasses="govuk-!-width-two-thirds")
+@showRows(item: String, value: BigDecimal, route: Call = controllers.employment.routes.CheckYourBenefitsController.show(taxYear, employmentId)) = @{
```

Comment:
I think this will be updated when the navigation is done, so instead of the carFuel question routing back to the check your benefits page, it will instead be routed to this amount page

Reply:
ok